src/components/Button/Button.tsx

```diff
@@ -18,9 +18,9 @@ const buttonVariants = cva(
 				danger: ""
 			},
 			size: {
-				sm: "px-1 h-6 text-text-sm",
-				md: "px-2 h-7 text-text-md",
-				lg: "px-4 h-8 text-text-lg"
+				sm: "px-2 h-6 text-text-sm text-sm",
+				md: "px-4 py-4 text-base h-7 text-text-md",
+				lg: "px-6 py-6 text-lg	h-8 text-text-lg"
```

Comment:
`text-lg` and `text-text-lg` are redundant I think. The we're using the one with `text-text` as we're also having `display` variants.